Shnadi (#39)
UtrechtUniversity/research-engineering: _pages/projects/index.md

### Current projects

| [Soothreat](https://github.com/UtrechtUniversity/patient-soothers-threats) | 
| --- | 
| **Time frame**: 12/2022 - present |
| **Research domain**: Clinical Psychology |
| **Technologies**: Python, Machine Learning, Natural Language Processing, Topic Modeling, LDA | 
| **Research Engineers**:  [Shiva Nadi](https://www.uu.nl/medewerkers/SNadiNajafabadi), [Raoul Schram](https://www.uu.nl/staff/RDSchram),  [Ken Krige] |

> | In this project we are developing scripts to help researchers to make a good understanding of what is considered as threats and soothers for people with a central sensitivity syndrome (i.e., irritable bowel syndrome, fibromyalgia, or chronic fatigue syndrome). Threats create experiences of danger, harm, damage, or unsafety and may worsen patients' physical symptoms.  Soothers on the other hand create feelings of calmness, well-being, safety, or social connectedness and that may alleviate patients' physical symptoms. We are dealing with multi topic short text in this project. |

Reviewer: ```suggestion
| In this project we are developing methods to help researchers to improve the understanding of threats and soothers for people with a central sensitivity syndrome (i.e., irritable bowel syndrome, fibromyalgia, or chronic fatigue syndrome). Threats create experiences of danger, harm, damage, or unsafety and may worsen patients' physical symptoms.  Soothers on the other hand create feelings of calmness, well-being, safety, or social connectedness and that may alleviate patients' physical symptoms. We are dealing with multiple topics and short texts in this project. |
```
